railgun/apis/configuration/v1beta1/tcpingress_types.go

```diff
@@ -72,35 +72,6 @@ type IngressTLS struct {
 	SecretName string `json:"secretName,omitempty"`
 }
 
-// IngressRule represents a rule to apply against incoming requests.
```

Comment:
Just curious, why we remove the TCPIngress rules ? do you want to make the ingress rule generic ?

Reply:
These were moved into a different file in the same package, to organize the `IngressRule` and the new `UDPIngressRule` together.